app/partials/collection-card.cjsx

```diff
@@ -64,6 +64,7 @@ module.exports = React.createClass
         <svg className="card-space-maker" viewBox="0 0 2 1" width="100%"></svg>
         <div className="details">
           <div className="name"><span>{@props.collection.display_name}</span></div>
+          {<div className="public-spaceholder">&nbsp;</div> if !@props.collection.private}
```

Comment:
I think this could be achieved with css alone and/or just have the private messaging display inline with the username.